src/test/java/org/springframework/data/couchbase/util/UnmanagedTestCluster.java

```diff
@@ -59,7 +59,7 @@ ClusterType type() {
 
 	@Override
 	TestClusterConfig _start() throws Exception {
-		bucketname = UUID.randomUUID().toString();
+		bucketname = "my_bucket"; // UUID.randomUUID().toString();
```

Comment:
Reuse "my_bucket" if it already exists.  Creating the indexes, scopes and collections takes time.

Reply:
There are some tests that don't delete everything they've inserted.  So I will remove this change from the changeset and leave it for later when I have had time to verify that all the tests clean up after themselves.